### ISSUE TITLE
[Blog] Gfycat embeds don't work in many old articles

### ISSUE THREAD

**Issue by wertercatt** [dolphin-emu/www#103]
Many blog articles use an old broken way of embedding Gfycat content, which sends completely broken html to the reader's browser.
Examples, using _Dolphin Progress Report: January 2016_ and _Dolphin Progress Report: July 2018_
Old method 1 (Dolphin Progress Report: January 2016):
`<p><center><iframe src="https://gfycat.com/iframe/ContentFrequentConch" frameborder="0" scrolling="no" width="320" height="240" style="-webkit-backface-visibility: hidden;-webkit-transform: scale(1);" ></iframe></center><center>Gotta go f-f-f-f-fast!</center></p>`
Old method 2 (Dolphin Progress Report: July 2018):
`<div class="embed-responsive embed-responsive-4by3"><div class="gfyitem" data-autoplay="true" data-id="ContentFrequentConch" data-responsive="true"></div></div>`
Neither of these methods currently work.
Gfycat recommends this method to embed content hosted on their site currently:
`<iframe src='https://gfycat.com/ifr/ContentFrequentConch' frameborder='0' scrolling='no' allowfullscreen width='640' height='524'></iframe><p> <a href="https://gfycat.com/contentfrequentconch">via Gfycat</a></p>`
But given how often they seem to change their embed method and break old ones, it is likely better to just migrate all the videos previously hosted on their site to the /m/user/blog/ directory tree like how it was done in _Dolphin Progress Report: December 2018 and January 2019_:
```
<div class="media-block full-width">
<figure>
<div class="embed-responsive embed-responsive-16by9">
<a href="https://dolphin-emu.org/m/user/blog/progress-report/2019-january/Android-touchscreen_full.mp4">
<video autoplay muted loop playsinline>
<source src="https://dolphin-emu.org/m/user/blog/progress-report/2019-january/Android-touchscreen.webm" type="video/webm"/>
<source src="https://dolphin-emu.org/m/user/blog/progress-report/2019-january/Android-touchscreen.mp4" type="video/mp4"/>
</video></div></a>
</figure>
</div>
```
A full list of links to blog article sections affected by this issue:
https://dolphin-emu.org/blog/2018/08/01/dolphin-progress-report-july-2018/#50-8324-50-8381-and-50-8412-reduce-netplay-bandwidth-usage-by-techjar
https://dolphin-emu.org/blog/2018/08/01/dolphin-progress-report-july-2018/#50-8443-improve-various-emulated-wii-remote-input-behaviors-by-iwubcode
https://dolphin-emu.org/blog/2018/08/01/dolphin-progress-report-july-2018/#configurable-motion-intensity
https://dolphin-emu.org/blog/2018/07/06/dolphin-progress-report-june-2018/#floating-up-to-nothing
https://dolphin-emu.org/blog/2018/04/02/dolphin-progress-report-february-and-march-2018/#presenting-a-change-to-shader-generation
https://dolphin-emu.org/blog/2018/02/03/dolphin-progress-report-january-2018/#memory-card-manager
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#how-dolphin-handled-xfb-usage
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#vertical-tiled-rendering
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#horizontal-tiled-rendering
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#extraincomplete-frames
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#store-xfb-copies-to-texture-and-ram
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#store-xfb-copies-to-texture-only
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#rendering-without-the-gpu_1
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#reading-old-frames_1
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#partial-xfb-copies
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#pace-yourself
https://dolphin-emu.org/blog/2017/11/19/hybridxfb/#immediately-present-xfb
https://dolphin-emu.org/blog/2016/04/01/dolphin-progress-report-march-2016/#40-9154-instruction-slice-timing-issue-fixes-by-phire
https://dolphin-emu.org/blog/2016/02/02/dolphin-progress-report-january-2016/#40-8586-poll-for-controller-input-at-the-correct-time-in-a-frame-by-booto
https://dolphin-emu.org/blog/2016/01/02/dolphin-progress-report-december-2015/#40-8340-video-interface-wait-until-start-of-active-video-lines-to-sample-external-frame-buffer-xfb-address-by-booto
https://dolphin-emu.org/blog/2015/11/01/dolphin-progress-report-october-2015/#40-7937-block-wx-input-when-detecting-controls-by-mickdermack
https://dolphin-emu.org/blog/2015/06/01/dolphin-progress-report-may-2015/#40-6415-preload-custom-textures-by-degasus
https://dolphin-emu.org/blog/2015/02/01/dolphin-progress-report-january-2015/#combating-zfighting-on-decals

Blog article sections using a currently proper method, listed here as I'm recommending completely cutting out the dependency on gfycat due to their disregard for embedrot:
https://dolphin-emu.org/blog/2018/08/14/state-of-android/#running-dolphin-on-android-in-2018
https://dolphin-emu.org/blog/2017/11/03/dolphin-progress-report-october-2017/#50-5688-do-not-allocate-gpu-texture-decoding-resources-on-systems-that-do-not-support-it-by-ligfx
https://dolphin-emu.org/blog/2017/11/03/dolphin-progress-report-october-2017/#50-5745-add-heuristic-for-detecting-custom-mipmaps-by-tomcc
https://dolphin-emu.org/blog/2017/10/02/dolphin-progress-report-september-2017/#50-5551-add-eject-disc-option-by-josjuice
https://dolphin-emu.org/blog/2017/09/02/dolphin-progress-report-july-and-august/#50-4869-ubershaders-20-by-stenzek
https://dolphin-emu.org/blog/2017/07/30/ubershaders/#the-problem
https://dolphin-emu.org/blog/2017/07/30/ubershaders/#asynchronous-shader-compilation
https://dolphin-emu.org/blog/2017/07/30/ubershaders/#ubershaders-20
https://dolphin-emu.org/blog/2016/09/01/dolphin-progress-report-august-2016/#50-480-allow-texture-samples-from-0x0-by-armada651
https://dolphin-emu.org/blog/2016/06/24/dolphin-50-release/#signal-corrected-output
https://dolphin-emu.org/blog/2016/06/24/dolphin-50-release/#exclusive-fullscreen
https://dolphin-emu.org/blog/2016/06/01/dolphin-progress-report-may-2016/#performance-queries
https://dolphin-emu.org/blog/2015/08/01/dolphin-progress-report-july-2015/#40-7133-automatically-detect-gamecube-aspect-ratios-by-phire
https://dolphin-emu.org/blog/2015/07/01/dolphin-progress-report-june-2015/#40-6704-optionally-emulate-accurate-nans-by-flacs
https://dolphin-emu.org/blog/2015/04/01/dolphin-progress-report-march-2015/#40-5788-enable-full-occlusion-queries-on-nexus-9-by-sonicadvance1
https://dolphin-emu.org/blog/2015/01/01/dolphin-progress-report-december-2014/#40-4889-enable-3d-stereoscopy-by-armada651
https://dolphin-emu.org/blog/2014/12/01/dolphin-progress-report-november-2014/#40-4222-support-constant-angular-velocity-for-disc-reads-by-josjuice
https://dolphin-emu.org/blog/2014/08/23/dolphinbar-review/#mouse-and-keyboard

**Comment by ExE-Boss** [comment]
I also reported this here: https://forums.dolphin-emu.org/Thread-to-the-screen-with-hybrid-xfb?pid=486581#pid486581

**Comment by ExE-Boss** [comment]
This is caused by: https://github.com/gfycat/gfycat.js/issues/25.

**Comment by MayImilae** [comment]
Yep yep, gfycat has been breaking things. And even then it's become annoying and troublesome even excluding it breaking. So I was working on moving us away from gfycat to fix this, but hit a snag. #99 should fix it. I'll push to get that merged.

**Comment by ExE-Boss** [comment]
I’m trying to fix `gfycat.js` in https://github.com/gfycat/gfycat.js/pull/28, but their **Travis CI** setup is the worst I’ve seen so far, and I’ve some rather horrible **CI** setups.

I’m trying to rectify that in https://github.com/gfycat/gfycat.js/pull/29.
